data/base-spells.yaml

```diff
@@ -56,6 +56,7 @@ prep_messages:
 - You produce an illusory coin and begin rolling it across your knuckles
```

Comment:
I know it isn't related to your change, but I think line 26 can be removed (it is redundant with line 32)